### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by ericwooley** [jaredhanson/passport-openidconnect#85]
The last commit was 4 years ago, there are 25 open issues, some of which are security related, and 24 pull requests pending.

Is this project dead? Or perhaps the code is just basically done?

Maybe it's time to add a new maintainer? 

I'm surprised that it isn't more widely used, since it seems to be the recommended package for openidconnect. Maybe I am missing something?

**Comment by JackPoint** [comment]
You can try to contact him on one of these ways: http://jaredhanson.net/

**Comment by sscarduzio** [comment]
do you know any valid alternative projects (implementing OIDC for passport, or express in general)?

**Comment by ericwooley** [comment]
Nope, we decided that SAML was enough, and we didn't need to implement OIDC.

**Comment by kastork** [comment]
There are many forks.  Hard to know if any should be candidates for replacing this.  The one at https://github.com/GovTechSG/passport-openidconnect looks promising, and the author publishes to npm. But it isn't working for me, seemingly due to being built for earlier versions of express and express-session.

**Comment by newmanw** [comment]
Wondering the same, anyone have any updates on project status or an alternative?
